main.js

```diff
@@ -3,73 +3,13 @@
 /* eslint-disable max-classes-per-file */
 /* eslint-disable no-trailing-spaces */
 /* eslint-disable lines-between-class-members */
```

Comment:
- [ ] Kindly avoid disabling linters. Linters help us write standardized code. Enable these linters and resolve any resulting issues.